test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -96,8 +100,7 @@ public CorfuReplicationReconfigurationIT(Pair<String, ExampleSchemas.ClusterUuid
     public static Collection<Pair<String, ExampleSchemas.ClusterUuidMsg>> input() {
 
         List<String> transportPlugins = Arrays.asList(
-                "src/test/resources/transport/grpcConfig.properties"
-//                "src/test/resources/transport/nettyConfig.properties"
+                "GRPC"
```

Comment:
Why we only add "GRPC" here? I wonder this since the comment says it tests for Netty and GRPC.

Reply:
for now, we are concentrating only on GRPC so applications are not blocked.
We will bring back netty soon. So instead of removing netty, I have it commented out.